cmd/util/ledger/migrations/accounts.go

```diff
@@ -0,0 +1,73 @@
+package migrations
+
+import (
+	"strings"
+
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func AddMissingKeysMigration(payloads []ledger.Payload) ([]ledger.Payload, error) {
```

Comment:
We want to perform the following migrations on **testnet**. We are still in the process of provisioning keys for the mainnet migration.

## Core Contracts

| Contract | Address | Encoded Account Key (including weight) |
|----------|---------|-----------------------------------------|
| FungibleToken | `9a0766d93b6608b7` | `f847b8402b0bf247520770a4bad19e07f6d6b1e8f0542da564154087e2681b175b4432ec2c7b09a52d34dabe0a887ea0f96b067e52c6a0792dcff730fe78a6c5fbbf0a9c02038203e8` |
| NonFungibleToken | `631e88ae7f1d7c20` | `f847b8402b0bf247520770a4bad19e07f6d6b1e8f0542da564154087e2681b175b4432ec2c7b09a52d34dabe0a887ea0f96b067e52c6a0792dcff730fe78a6c5fbbf0a9c02038203e8` |
| FlowToken | `7e60df042a9c0868` | `f847b8402b0bf247520770a4bad19e07f6d6b1e8f0542da564154087e2681b175b4432ec2c7b09a52d34dabe0a887ea0f96b067e52c6a0792dcff730fe78a6c5fbbf0a9c02038203e8` |
| FlowFees | `912d5440f7e3769e` | `f847b8402b0bf247520770a4bad19e07f6d6b1e8f0542da564154087e2681b175b4432ec2c7b09a52d34dabe0a887ea0f96b067e52c6a0792dcff730fe78a6c5fbbf0a9c02038203e8` |
| Staking | `9eca2b38b18b5dfe` | `f847b8402b0bf247520770a4bad19e07f6d6b1e8f0542da564154087e2681b175b4432ec2c7b09a52d34dabe0a887ea0f96b067e52c6a0792dcff730fe78a6c5fbbf0a9c02038203e8` |


## Non-Core Contracts

**NOTE:** Key is different than above

| Contract | Address | Encoded Account Key (including weight) |
|----------|---------|-----------------------------------------|
| StakingProxy | `7aad92e5a0715d21` | `f847b840a272d78cfa14eb248d95c12da8c6a24db9fda5ceddc07444080b49ef6cd15a06e88223af8acd235e2ff7a627adb81cf37d0a1384d9985de4bc7a7fc7eb86848402038203e8` |
| LockedTokens | `95e019a17d0e23d7` | `f847b840a272d78cfa14eb248d95c12da8c6a24db9fda5ceddc07444080b49ef6cd15a06e88223af8acd235e2ff7a627adb81cf37d0a1384d9985de4bc7a7fc7eb86848402038203e8` |

Reply:
I added these @psiemens could you please review them one more time in the code